sklearn/ensemble/forest.py

```diff
@@ -437,16 +437,12 @@ def _validate_y_class_weight(self, y):
         y = y_store_unique_indices
 
         if self.class_weight is not None:
-            valid_presets = ('auto', 'balanced', 'balanced_subsample', 'subsample', 'auto')
```

Comment:
this part needs to be undone. It should be 0.19. Please change the error accordingly. Why is auto in this list twice?